cmake/modules/platform/toolcfg/xlc.cmake

```diff
@@ -96,3 +96,6 @@ list(APPEND TR_C_COMPILE_OPTIONS
 
 set(SPP_CMD ${CMAKE_C_COMPILER})
 set(SPP_FLAGS -E -P)
+
+#Configure the platform dependent library for multithreading
+set(THREAD_LIBRARY -lpthread)
```

Comment:
This should actually be just `pthread`. I believe the original code's `-l` was a typo.